cmd/vfkit/main_test.go

```diff
@@ -42,3 +44,66 @@ func TestStartIgnitionProvisionerServer(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, ignitionData, body)
 }
+
+func TestGenerateCloudInitImage(t *testing.T) {
+	_, filename, _, ok := runtime.Caller(0)
+	assert.True(t, ok)
+
+	currentDir := filepath.Dir(filename)
+	projectRoot := filepath.Join(currentDir, "../../")
```

Comment:
can this be moved to a `getTestAssetsDir` as it's reused by all the tests? An alternative could be to add a `testdata` directory to `cmd/vfkit` which should be easier to locate?
Did you try using https://pkg.go.dev/os#Getwd instead of `runtime.Caller`?

Reply:
i was not sure if adding a folder in cmd/vfkit was the right way to do it in golang, this is why I kept the assets in the test folder. I applied your suggestions but if you prefer to move the files I can do that

Reply:
I'm also not sure about the recommended way of doing this, let's keep it as it is for now, we can move this around if we are more knowledgeable one day :)